docs/src/What's New.doc.js

```diff
@@ -0,0 +1,74 @@
+// @flow strict
+import React, { type Node, useEffect } from 'react';
+import { Box, Text, Link, Stack, Row } from 'gestalt';
+import Markdown from './components/Markdown.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+function Changelog() {
+  const [changelogData, setChangelogData] = React.useState('Loading...');
+
+  const getChangelog = () => {
+    fetch('https://cdn.jsdelivr.net/gh/pinterest/gestalt@master/CHANGELOG.md')
+      .then(response => {
+        if (!response.ok) {
+          return '# There was error loading the changelog, please try again later.';
+        }
+        return response.text();
+      })
+      .then(data => {
+        setChangelogData(data);
+      });
+  };
+
+  useEffect(() => {
+    getChangelog();
```

Comment:
@ayeshakmaz could we use async / await and inline the function since we don't use it anywhere else?

```
useEffect(() => {
    const fetchChangelog = async () => {
      const result = await fetch('https://cdn.jsdelivr.net/gh/pinterest/gestalt@master/CHANGELOG.md');
      setChangelogData(response.ok ? await response.text() : '# There was error loading the changelog, please try again later.');
    };
 
    fetchChangelog();
}, []);
```

Reply:
OO yes, sure thing